core/jobs.py

```diff
@@ -1134,24 +1389,26 @@ def get_continuous_computations_info(cc_classes):
     """Returns data about the given computations.
 
     Args:
-      cc_classes: a list of subclasses of BaseContinuousComputationManager.
+        cc_classes: list(BaseContinuousComputationManager). List of subclass
+            computations.
 
     Returns:
-      A list of dicts, each representing a continuous computation. Each dict
-      has the following keys:
-      - 'computation_type': the type of the computation
-      - 'status_code': the current status of the computation
-      - 'last_started_msec': when a batch job for the computation was last
-            started, in milliseconds since the epoch
-      - 'last_finished_msec': when a batch job for the computation last
-            finished, in milliseconds since the epoch
-      - 'last_stopped_msec': when a batch job for the computation was last
-            stopped, in milliseconds since the epoch
-      - 'active_realtime_layer_index': the index of the active realtime layer
-      - 'is_startable': whether an admin should be allowed to start this
-            computation
-      - 'is_stoppable': whether an admin should be allowed to stop this
-            computation
+        A list of dicts, each representing a continuous computation using the
```

Comment:
list(dict). Each dict represents a continuous computation and has the following keys: